tests/gtests/sw_testhelpers.cc

```diff
@@ -15,18 +15,22 @@
 #include <time.h>
 #include <unistd.h>
 
-// externs `*logfp`, `errstr`, `logged`, `QuietMode`, `EchoInits`
 #include "include/generic.h"
-#include "include/filefuncs.h" // externs `_firstfile`, `inbuf`
+#include "include/filefuncs.h"
 #include "include/SW_Site.h"
 #include "include/SW_SoilWater.h"
 #include "include/SW_Weather.h"
 #include "include/SW_Control.h"
-#include "include/SW_Main_lib.h" // for `sw_check_log()`
+#include "include/SW_Files.h"
+#include "include/myMemory.h"
 
 #include "tests/gtests/sw_testhelpers.h"
 
+SW_OUTPUT_POINTERS SW_OutputPtrs;

```

Comment:
Why are some of the gtest global variables defined in `sw_maintest.cc` and others in `sw_testhelpers.cc`?